tests/EscapeCssTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+use JakubBoucek\Escape\Escape;
+use JakubBoucek\Escape\EscapeCss;
+use Tester\Assert;
+use Tester\Environment;
+use Tester\TestCase;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+Environment::setup();
+
+/** @testCase */
+class EscapeCssTest extends TestCase
+{
+    public function getCssColorArgs(): array
+    {
+        return [
+            ['darkblue', 'darkblue'],
+            ['#000', '#000'],
+            ['#00008b', '#00008b'],
+            ['#00008bff', '#00008bff'],
+            ['#00008BFF', '#00008BFF'],
+            ['rgb(0,0,139)', 'rgb(0,0,139)'],
+            ['rgba(0, 0, 139, 0.8)', 'rgba(0, 0, 139, 0.8)'],
+            ['rgba(0, 0, 139 / 0.8)', 'rgba(0, 0, 139 / 0.8)'],
+            ['hsl(240,100%,27%)', 'hsl(240,100%,27%)'],
+            ['hsla( 240, 100%, 27%, 0.8)', 'hsla( 240, 100%, 27%, 0.8)'],
+            ['hsla( 240, 100%, 27% / 0.8)', 'hsla( 240, 100%, 27% / 0.8)'],
+            ['ActiveText', 'ActiveText'],
+            ['lab(29.2345% 39.3825 20.0664)', 'lab(29.2345% 39.3825 20.0664)'],
+            ['lab(52.2345% 40.1645 59.9971 / .5)', 'lab(52.2345% 40.1645 59.9971 / .5)'],
+            ['lch(52.2345% 72.2 56.2)', 'lch(52.2345% 72.2 56.2)'],
+            ['lch(52.2345% 72.2 56.2 / .5)', 'lch(52.2345% 72.2 56.2 / .5)'],
+            [' darkblue', 'darkblue'],
```

Comment:
Oddělil bych do druhého `DataProvideru` hodnoty které budou změněné, a v prvním bych nechal ty, co se nezmění. Zvětší se tím přehlednost, hned bude jasno, co funkce dělá a jak zpracovává různé druhy dat. `PHPUnit` umí používat několik `DataProvideru` najednou. Asi Tester by to také měl zvládnout.
[Dokumentace](https://phpunit.readthedocs.io/en/9.5/writing-tests-for-phpunit.html#writing-tests-for-phpunit-data-providers-examples2-datatest-php)

Reply:
Rozumím, ale to Tester nepodporuje. Upravil jsem to alespoň rozdělením bloků.

Reply:
super, takové řešení mě ani nenapadlo